### PR TITLE
task_DEV-2

### PR DESCRIPTION
**Pull request by DmitryZapryvaryn**

### CODE REVIEW

File: Task_02/EvenChar.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Text;
+
```
Comment (IgorKheidorov) [inline Task_02/EvenChar.cs:3]: Empty lines

File: Task_02/EvenChar.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    class EvenChar
+    {
+        /// <summary>
+        /// Method finds even char from input sring.
+        /// </summary>
+        /// <param name="inputString">Character sequance.</param>
+        /// <returns>
+        /// Substring contains even char from input string.
+        /// </returns>
+        public StringBuilder getEvenCharSequance(string inputString)
```
Comment (IgorKheidorov) [inline Task_02/EvenChar.cs]: Code style violation, grammar error in name

File: Task_02/EvenChar.cs [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    /// </summary>
+    class EvenChar
+    {
+        /// <summary>
+        /// Method finds even char from input sring.
+        /// </summary>
+        /// <param name="inputString">Character sequance.</param>
+        /// <returns>
+        /// Substring contains even char from input string.
+        /// </returns>
```
Comment (IgorKheidorov) [inline Task_02/EvenChar.cs]: Bad name, you returns builder but not sequence

File: Task_02/EvenChar.cs [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        public StringBuilder getEvenCharSequance(string inputString)
+        {
+            StringBuilder result = new StringBuilder();
+            if (inputString.Length == 0)
+            {
+                throw new ArgumentNullException();
```
Comment (IgorKheidorov) [inline Task_02/EvenChar.cs]: not processed exception

File: Task_02/EvenChar.cs [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+            }
+            else
+            {
+                for(int i = 0; i < inputString.Length; i++)
+                {
+                    if( i % 2 == 0 )
```
Comment (IgorKheidorov) [inline Task_02/EvenChar.cs]: Why do you calculate index each time? better use i+=2

File: Task_02/EvenChar.cs [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+                    {
+                        result.Append(inputString[i]);
+                    }
+                }
+            }
+            return result;
+        }
+
+        //Only for test program
```
Comment (IgorKheidorov) [inline Task_02/EvenChar.cs]: Can't understand comment

File: Task_02/EvenChar.cs [hunk 7]
```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Text;
+
+
+namespace Task_02
+{
+    /// <summary>
+    /// This class contains method which find substring with even characters
+    /// </summary>
+    class EvenChar
```
Comment (IgorKheidorov) [inline Task_02/EvenChar.cs]: Bad unclear class name, reading the name I suppose that this is even chars, but not sequence or class to process something

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]

File: Task_02/EvenChar.cs [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+{
+    /// <summary>
+    /// This class contains method which find substring with even characters
+    /// </summary>
+    class EvenCharStringBuilder
+    {
+        /// <summary>
+        /// Method finds even char from input sring.
+        /// </summary>
+        /// <param name="inputString">Character sequence.</param>
```
Comment (AlexanderSoroka) [inline Task_02/EvenChar.cs]: invalid argument description

File: Task_02/EvenChar.cs [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+        public StringBuilder GetEvenCharSequence(string inputString)
+        {
+            StringBuilder result = new StringBuilder();
+            for(int i = 0; i < inputString.Length; i += 2)
+            {
+                result.Append(inputString[i]);
+            }
+            return result;
+        }
+        //Entry point
```
Comment (AlexanderSoroka) [inline Task_02/EvenChar.cs]: useless comment, missed blank line

**Review by AlexanderSoroka — CHANGES_REQUESTED** [review]

**Review by AlexanderSoroka — APPROVED** [review]

**Pull request merged**